lib/internal/test_runner/coverage.js

```diff
@@ -177,6 +187,13 @@ class TestCoverage {
         if (j > 0 && ranges.length > 0) {
           const range = ranges[0];
 
+          ArrayPrototypePush(functionReports, {
+            __proto__: null,
+            name: functions[j].functionName,
+            count: MathMax(...ArrayPrototypeMap(ranges, (r) => r.count)),
```

Comment:
Ii looks like you recomputing the maximum number more than needed - this is in a loop of every `ranges` item the max count should not change between loop runs from what I can tell